libkbfs/file_data.go

```diff
@@ -65,13 +65,21 @@ type parentBlockAndChildIndex struct {
 	childIndex int
 }
 
+func (pbci parentBlockAndChildIndex) childIPtr() IndirectFilePtr {
+	return pbci.pblock.IPtrs[pbci.childIndex]
+}
+
+func (fd *fileData) rootBlockPointer() BlockPointer {
```

Comment:
@jzila specifically requested this as part of this PR.  `childIPtr()` in particular makes a lot of sense as part of this PR, I think it would make the PR harder to review since it simplifies a lot of the new code you'd look at in the new passes.  I don't care as much about `rootBlockPointer()`, but it seems like it should be fairly easy to review the existing lines that only have that call changed.  Do you feel strongly about it?  How would splitting it off help exactly?